docs/prepare_prior_versions.py

```diff
@@ -534,20 +534,15 @@ def prepare_prior_version(version: Version) -> None:
     if version >= Version(0, 16):
         prepend_version_info_to_name_for_md_images(version)
 
-    # TODO: delete this after we cut 0.18.9
-    if version >= Version(0, 18) and version < Version(0, 18, 9):
-        remove_extension_from_absolute_md_links(version)
-
 
 @total_ordering
 @dataclass(frozen=True)
 class Version:
     major: int = 0
     minor: int = 0
-    patch: int = 0
 
-    def as_tuple(self) -> tuple[int, int, int]:
-        return self.major, self.minor, self.patch
+    def as_tuple(self) -> tuple[int, int]:
+        return self.major, self.minor
```

Comment:
Sorry just seeing this code now - could we instead just leverage `version.Version`? Feels like a place where the stdlib has what we need